account-kit/react-native/src/index.ts

```diff
@@ -19,6 +19,9 @@ export {
   useSignMessage,
   useSignTypedData,
   useSmartAccountClient,
+  useSolanaConnection,
+  useSolanaSigner,
+  useSolanaTransaction,
```

Comment:
Not sure if there is a reason we never exported these?  In my testing it seems to work